src/calendar/date/CalendarEventViewModel.ts

```diff
@@ -168,6 +170,7 @@ export class CalendarEventViewModel {
 		existingEvent: CalendarEvent | null,
 		responseTo: Mail | null,
 		resolveRecipientsLazily: boolean,
+		updateViewCallback: () => void = noOp
```

Comment:
I generally like this pattern but this VM wasn't written with it and in this case it would be easier to listen for `initialized()` in the veiw and do redraw there?

Reply:
I decided to await initialized in the createCalendarEventViewModel function which is already async and always used to create a CalendarEventViewModel. What do you think about this?

Reply:
I think this works. It will delay the dialog by a bit tho for loading alarms but it should be negligible.